src/Protocol/ProtocolRecognizer.h

```diff
@@ -53,7 +53,11 @@ class cProtocolRecognizer :
 	static AString GetVersionTextFromInt(int a_ProtocolVersion);
 
 	/** GetPacketId is implemented in each protocol version class */
-	virtual UInt32 GetPacketId(eOutgoingPackets a_Packet) override { return 0; }
+	virtual UInt32 GetPacketId(eOutgoingPackets a_Packet) override
+	{
+		ASSERT(!"cProtocolRecognizer::GetPacketId should never be called! Something is horribly wrong! (it implies that someone other than a Protocol-derived class is calling GetPacketId)");
```

Comment:
I think `it implies` is wrong here - it should be `this implies`.

Reply:
Maybe `this method being called implies`?